Duo/_meta/manifest.yml

```
@@ -0,0 +1,5 @@
+uuid: be246024-5e6b-4df0-83dc-bd61226fbffd
+name: Duo [BETA]
```

Beta flag is not required for module name
```suggestion
name: Duo
```